### PR TITLE
simple CNN following-ish the pytorch CIFAR-10 tutorial

### PR DESCRIPTION
**Pull request by jonahkrop**
Pretty simple CNN architecture following the [cifar-10](https://pytorch.org/tutorials/beginner/blitz/cifar10_tutorial.html#training-on-gpu) tutorial, but expanded to fit our larger 256x256 image sizes

as a result, added a couple more convolutional layers and a larger fully connected layer at the end. Getting loss down from 2.3 to 1.8 or so at the moment

### CODE REVIEW

File: simple_cnn.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+        # forward + backward + optimize
+        outputs = net(inputs)
+        loss = criterion(outputs, labels)
+        loss.backward()
+        optimizer.step()
+
+        # print statistics every 100 batches
+        running_loss += loss.item()
+        j = 10000 / batch_size / 10
```
Comment (cpak10) [inline simple_cnn.py]: We might want to convert this to `j = 10_000 / batch_size // 10` just incase we want to change batch sizes in the future. But otherwise looks great!

**Review by cpak10 — APPROVED** [review]

File: simple_cnn.py (hunk 1 above)
Comment (jonahkrop) [reply simple_cnn.py]: excellent suggestion
